### ISSUE TITLE
API (CLI) to minimize Termux:Float

### ISSUE THREAD

**Issue by mentalisttraceur** [termux/termux-float#52]
It would be really great if there was a `termux-float-minimize` command which programmatically did what the button in the top left does: tell Termux:Float to minimize.

I want this because it would be *much* more ergonomic and fast for me to hit keys on my keyboard where my thumbs already are than to move/stretch my hands or fingers up to the top of my floating Termux - especially when floating Termux is fairly large, so for example:

1. I would bind a key in Emacs to execute this command (and Emacs is my shell, terminal multiplexer, and so on, and I can do everything I might do in Termux through Emacs, so this would basically give me a universal super fast super ergonomic way to switch out of Termux:Float), and
2. I would write various functions/scripts to end their actions by minimizing Termux, because there are some workflows/actions after which I often or even *always* want to switch out of Termux.

In fact, I think this would be so huge for me that it would single-handedly flip my usage from almost entirely regular Termux to almost entirely floating Termux.

**Comment by mentalisttraceur** [comment]
~~This is another feature which I would be super happy to do the work to try to implement and PR if the idea isn't rejected.~~

[Edit: I stopped needing/using/wanting floating Termux sometime in 2024. I think this could still be very useful -- enables making termux-float a more ergonomic quick-access control panel -- but I no longer have the drive to make this happen for the foreseeable future.]

**Comment by mentalisttraceur** [comment]
Of course, if I wasn't using Emacs for everything, I would bind keys in zsh, Neovim, and tmux to call this command, giving me almost the same coverage for always being able to super ergonomically switch out of a large floating Termux.

**Comment by muyiboysaka123** [comment]
[]()
